spec.html

```diff
@@ -39663,6 +39824,31 @@ <h1>%TypedArray%.prototype.values ( )</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-%typedarray%.prototype.with">
+        <h1>%TypedArray%.prototype.with ( _index_, _value_ )</h1>
+        <p>This method performs the following steps when called:</p>
+        <emu-alg>
+          1. Let _O_ be the *this* value.
+          1. Perform ? ValidateTypedArray(_O_).
+          1. Let _len_ be _O_.[[ArrayLength]].
+          1. Let _relativeIndex_ be ? ToIntegerOrInfinity(_index_).
+          1. If _relativeIndex_ ≥ 0, let _actualIndex_ be _relativeIndex_.
+          1. Else, let _actualIndex_ be _len_ + _relativeIndex_.
+          1. If _O_.[[ContentType]] is ~BigInt~, set _value_ to ? ToBigInt(_value_).
+          1. Else, set _value_ to ? ToNumber(_value_).
```

Comment:
These steps overwrite the `_value_` parameter, which is somewhat discouraged. 
```suggestion
          1. If _O_.[[ContentType]] is ~BigInt~, let _numericValue_ be ? ToBigInt(_value_).
          1. Else, let _numericValue_ be ? ToNumber(_value_).
```
and then tweak the subsequent reference to `_value_`.

Reply:
Done. Much nicer.